lib/cmd/fhc/target.js

```diff
@@ -56,6 +56,7 @@ function target (argv, cb) {
     function (cb){
       //First, checking if the target platform version is greater than a minimum value.
       version.checkTargetVersion([tar], cb);
+      fhc.config.set("fhversion", 3);
```

Comment:
I am not sure we should blindly apply version 3 here. What if the target platform is still on version 2?

Instead, I think we should add this in the implementation of `version.checkTargetVersion`. Because in there we will know for sure that the target platform is on version 3, and then we can set this flag.

@camilamacedo86 does it make sense?